packages/core/src/codewhispererChat/controllers/chat/messenger/messenger.ts

```diff
@@ -306,12 +306,28 @@ export class Messenger {
                             }
                             const tool = ToolUtils.tryFromToolUse(toolUse)
                             if ('type' in tool) {
+                                let explanation: string | undefined = undefined
                                 let changeList: Change[] | undefined = undefined
                                 let messageIdToUpdate: string | undefined = undefined
                                 const isReadOrList: boolean = [ToolType.FsRead, ToolType.ListDirectory].includes(
                                     tool.type
                                 )
-                                if (tool.type === ToolType.FsWrite) {
+                                if (tool.type === ToolType.ExecuteBash) {
```

Comment:
combine them? `if (tool.type === ToolType.ExecuteBash || tool.type === tool.type === ToolType.FsWrite) `

Reply:
cannot do this easily because FsWrite has some additional logic.

Also need to cast the input as ExecuteBashParams or FsWriteParams based on the tool